Resources/services/export_fields.xml

```diff
@@ -9,6 +9,11 @@
             <bind key="$router" id="router" type="service" />
         </defaults>
 
+        <service id="OmikronFactfinder\Components\Data\Article\FieldProvider">
+            <argument type="tagged" tag="factfinder.export.field" />
+            <argument>%omikron_factfinder.export.articles.columns%</argument>
```

Comment:
perhaps we can change name to `omikron_factfinder.export.articles.columns_base `or something with similar meaning to stress that these are not all the columns which module is exporting by default.